cypress/integration/group2/notificationBadge.ts

```diff
@@ -0,0 +1,108 @@
+/*
+ *    Copyright 2018 OICR
```

Comment:
Might as well replace this with an up-to-date copyright block (the one that mentions "OICR", "UCSC", and "2022").

Reply:
oops forgot to add this in my recent commit. will add  before merging if all else is good